laser_encoders/test_laser_tokenizer.py

```diff
@@ -0,0 +1,101 @@
+import os
```

Comment:
Can you please copy paste the file header from `laser_tokenizer.py` here as well before merging?

Reply:
done

Reply:
Great!